exercises/flatten-array/FlattenArrayExample.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+func flattenArray(list: [Any?]) -> [Any] {
```

Comment:
This is a actually a difficult problem to solve well while keeping strong typing. I would recommend constraining to  Int Array and 2 levels of nesting arrays.